docpages/example_programs/misc/checking-member-permissions.md

```diff
@@ -80,8 +84,6 @@ bot.on_interaction_create([](const dpp::interaction_create_t& event) {
 });
 ```
 
-\note When using default permissions you don't necessarily need to check the issuing user for any permissions in the interaction event as Discord handles all that for you. But if you'd sleep better...
```

Comment:
why remove this? It is still true...

Reply:
because it's not always unnecessary. people do it when they dont want admins to override the permissions. But i guess you're right. I've put it to the top now, thought it would fit there more. Is that ok?